enclave-runtime/src/lib.rs

```diff
@@ -578,6 +582,79 @@ where
 	Ok(())
 }
 
+/// For now get the DOT/currency exchange rate from coingecko API.
+#[no_mangle]
+pub unsafe extern "C" fn update_market_data_xt(
+	genesis_hash: *const u8,
+	genesis_hash_size: u32,
+	currency: *const u8,
+	currency_size: u32,
+	unchecked_extrinsic: *mut u8,
+	unchecked_extrinsic_size: u32,
+) -> sgx_status_t {
+	let genesis_hash_slice = slice::from_raw_parts(genesis_hash, genesis_hash_size as usize);
+	let genesis_hash = hash_from_slice(genesis_hash_slice);
+
+	let mut curr_slice = slice::from_raw_parts(currency, currency_size as usize);
+	let curr: String = Decode::decode(&mut curr_slice).unwrap();
+
+	let xts = match update_market_data_internal(genesis_hash, curr) {
```

Comment:
`xt` is also somewhat of a gray area - probably `extrinsics` would be better as well. A new developer won't know that `xt` = `extrinsic`

Reply:
Here again xt, xts are used for extrinsic (s) throughout the code. Even in function names. I change it here, but we should apply theses changes all over the code.